lib/pronounce/phone_type.rb

```diff
@@ -4,37 +4,33 @@
 
 module Pronounce
   class PhoneType
-    SHORT_VOWELS = %w[AE AH EH IH UH]
+    SHORT_VOWELS = %w(AE AH EH IH UH)
 
     extend Forwardable
     include Comparable
 
     class << self
       def [](name)
-        if types.has_key? name
+        if types.key? name
           types[name]
         else
-          raise ArgumentError.new('invalid name')
+          fail ArgumentError, 'invalid name'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.